test/functional/connection-options-reader/connection-options-reader.ts

```diff
@@ -11,7 +11,7 @@ async function createDotenvFiles() {
 }
 
 async function createYamlFiles() {
-  await fs.mkdir(path.join(__dirname, "configs/yaml"));
+  await fs.mkdir(path.join(__dirname, "configs/yaml"), { recursive: true });
```

Comment:
Why is this change needed?

Reply:
I wasn't able to run test-fast multiple times in a row. It would complain that the directory already existed.

Reply:
That's a bigger problem that our tests have than just this.  Can you omit this change for now just to simplify this PR & what it touches?  If we have to revert or find the cause of something it makes it a whole lot easier if it's one PR for one change.